docs/machine-learning/tutorials/snippets/text-classification-tf/csharp/Program.cs

```diff
@@ -1,191 +1,179 @@
-        {
-            [VectorType(2)]
-            public float[] Prediction { get; set; }
-        }
-        // </SnippetPrediction>
-
-        // <SnippetVariableLengthFeatures>
-        /// <summary>
-        /// Class to hold the variable length feature vector. Used to define the
-        /// column names used as input to the custom mapping action.
-        /// </summary>
-        public class VariableLength
-        {
-            /// <summary>
-            /// This is a variable length vector designated by VectorType attribute.
-            /// Variable length vectors are produced by applying operations such as 'TokenizeWords' on strings
-            /// resulting in vectors of tokens of variable lengths.
-            /// </summary>
-            [VectorType]
-            public int[] VariableLengthFeatures { get; set; }
-        }
-        // </SnippetVariableLengthFeatures>
-
-        // <SnippetFixedLengthFeatures>
-        /// <summary>
-        /// Class to hold the fixed length feature vector. Used to define the
-        /// column names used as output from the custom mapping action,
-        /// </summary>
-        public class FixedLength
-        {
-            /// <summary>
-            /// This is a fixed length vector designated by VectorType attribute.
-            /// </summary>
-            [VectorType(FeatureLength)]
-            public int[] Features { get; set; }
-        }
-        // </SnippetFixedLengthFeatures>
-    }
+// <SnippetMovieReviewClass>
+/// <summary>
+/// Class to hold original sentiment data.
+/// </summary>
+public class MovieReview
+{
+    public string ReviewText { get; set; }
+}
+//</SnippetMovieReviewClass>
+
+//<SnippetPrediction>
+/// <summary>
+/// Class to contain the output values from the transformation.
+/// </summary>
+public class MovieReviewSentimentPrediction
+{
+    [VectorType(2)]
+    public float[] Prediction { get; set; }
+}
+// </SnippetPrediction>
+
+// <SnippetVariableLengthFeatures>
+/// <summary>
+/// Class to hold the variable length feature vector. Used to define the
+/// column names used as input to the custom mapping action.
+/// </summary>
+public class VariableLength
+{
+    /// <summary>
+    /// This is a variable length vector designated by VectorType attribute.
+    /// Variable length vectors are produced by applying operations such as 'TokenizeWords' on strings
+    /// resulting in vectors of tokens of variable lengths.
+    /// </summary>
+    [VectorType]
+    public int[] VariableLengthFeatures { get; set; }
+}
+// </SnippetVariableLengthFeatures>
+
+// <SnippetFixedLengthFeatures>
+/// <summary>
+/// Class to hold the fixed length feature vector. Used to define the
+/// column names used as output from the custom mapping action,
+/// </summary>
+public class FixedLength
+{
+    /// <summary>
+    /// This is a fixed length vector designated by VectorType attribute.
+    /// </summary>
+    [VectorType(FeatureLength)]
```

Comment:
An error here is the only one with the snippets build: 
```
error CS8801: Cannot use local variable or local function 'FeatureLength' declared in a top-level statement in this context
```

I'm not quite sure how to fix this.

Reply:
@jwood803 Not sure if this is the most elegant solution, but having any config / constant values in a static class works. I'll dig into it more.

```csharp
static class Config
{
    public const int FeatureLength = 600;
}

class FixedLength
{
    [VectorType(Config.FeatureLength)]
    public int[] Features { get; set; }
}
```

Reply:
In the meantime, we can go with this solution though

Reply:
Thanks! Updated the snippet and added the info in the tutorial